src/act.core.web/Services/INotifier.cs

```diff
@@ -115,14 +115,17 @@ private async Task NotifyNodeFailureByChefId(AutomateWebHookMessage message)
                 {
                     p.Fqdn,
                     p.Owner,
-                    p.BuildSpecificationId
+                    p.BuildSpecificationId,
+                    p.RemedyGroupEmailList
                 }).FirstOrDefaultAsync();
 
 
             if (result != null)
             {
                 var name = result.Owner.OwnerText(false);
-                var emails = new List<string> {result.Owner.Email};
+                var emails = !string.IsNullOrEmpty(result.RemedyGroupEmailList)
+                    ? new List<string> {result.Owner.Email, result.RemedyGroupEmailList.ToLower()}
+                    : new List<string> {result.Owner.Email};
```

Comment:
We seem to make a point to call .ToLower() when adding some emails to the list, but not for result.Owner.Email.  Is there a reason we don't use .ToLower() on that one?
We also compose email lists in a similar way in other files above, but we aren't using .ToLower() on those.
Should we be consistent when processing the email addresses?

Reply:
Updated other Email Addresses to ToLower() in this file. We only need to do .ToLower() here because we are fetching distinct email addresses here. But for other files this is not the case.